.github/Makefile

```diff
@@ -0,0 +1,116 @@
+SHELL := /bin/bash
```

Comment:
Should we put all of the macOS helpers in something like `.github/macos-installer/` to help clarify that we only need it for a subset of work?

(I haven't looked far enough to see if we need this for other platforms at all.)

Reply:
Absolutely! That will help clarify components that are only needed for macOS. I'll leave the scripts we also use for Linux where they are, though.